### PR TITLE
Banner, Navbar, Footer

### PR DESCRIPTION
**Pull request by danielwlemmon**
I created and did initial styling of the next event banner, NavBar, Footer.  In addition, I created the privacy policy route and added the navbar and footer to other existing routes/pages.

### CODE REVIEW

File: app/components/event_banner.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+//need to load next event data and have banner link to single event page (slug)
+
+export default function EventBanner() {
+  const banner_style = {
+    color: "white",
+    backgroundColor: "#A379C9",
+    padding: "10px",
+    textAlign: "center",
```
Comment (JeffBaumgardt) [inline app/components/event_banner.tsx:10]: Lets look into styled components or use Chakra Box. https://chakra-ui.com/docs/components/box

If you install Chakra then we can start wrapping things up in a theme https://chakra-ui.com/docs/styled-system/customize-theme

This will give us branding colors to set in a global rather than copy pasta in single components.

File: app/components/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import { Link } from "@remix-run/react";
+import PrivacyPolicy from "~/routes/privacy_policy";
+
+import ReactDenverLogo from "../assets/react_denver_logo.svg";
+import YouTube from "../assets/YouTube.svg";
+import Discord from "../assets/Discord.svg";
+import GitHub from "../assets/GitHub.svg";
+import Twitter from "../assets/Twitter.svg";
```
Comment (JeffBaumgardt) [inline app/components/footer.tsx:8]: Have a look at Remix's directory import paths. I think by default it's `~/` so instead of going relative path down to root and into assets it would look more like `import YouTube from "~/assets/YouTube.svg"` Just incase you ever want to move a file around and you don't get into a relative path nightmare. Ie. `../../../thing/thing2/foo.tsx`

File: app/components/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      </a>
+      <a href="https://github.com/reactdenver">
+        <img style={{ maxHeight: "50px" }} src={GitHub} />
+      </a>
+      <a href="https://mobile.twitter.com/ReactDenver">
+        <img style={{ maxHeight: "50px" }} src={Twitter} />
+      </a>
+      <a href="https://www.youtube.com/c/ReactDenver">
+        <img style={{ maxHeight: "50px" }} src={YouTube} />
+      </a>
```
Comment (JeffBaumgardt) [inline app/components/footer.tsx:33]: Probably could use a css file and a className attribute. ``

Then a css file that you can import along with the other imports in the same file
`import './footerStyles.css'`

```css
.brandingIcon {
  maxHeight: 50px;
}
```

File: app/components/footer.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      <a href="">Netlify</a>
+      <Link
+        style={{
+          padding: "0 5px 0 10px",
+          color: "#666666",
+          textDecoration: "none",
+        }}
+        to="/privacy_policy"
+      >
```
Comment (JeffBaumgardt) [inline app/components/footer.tsx:42]: We could get better styles out of Charka Link here - https://chakra-ui.com/docs/components/link

All if Charka's components have a prop `as` that allows you to modify the parent component in fun ways. For instance you can use the Charka Link with a Remix Link in the as to get all the goodness out of remix's link, like the to prop.

```jsx
import {Link as RouterLink} from "@remix-run/react"
import {Link} from "@chakra-ui/react"

return (
  <Link as={RouterLink} p={[0, 1, 0, 2.5]} to="/privacy_policy">Privacy Policy</Link>
)
```

Padding, margins etc can all be found - https://chakra-ui.com/docs/styled-system/style-props#margin-and-padding

Basically in my example above I have `p` == `padding` and I pass an array of the 4 values for padding. There's also `px` which is just the left, right and `py` which is top, bottom. But there is also `pt, pr, pb, pl` should you wish to go graniumar, such as overwriting just the right padding for example.

**Comment by JeffBaumgardt** [comment]
So far it looks really good. A nice place to start. This week we can start on more components and that will help solidify how to do styles.

@michaeldharrington what's your opinion of styling components in Chakra? I'm not sure how much in-love I am with how they are just extending the types of CSSProperties into the base components. We might want to utilize styled-components, emotion, or tailwind. 

Myself coming from MUI they have embraced emotion/styled-components for major styles and the `sx` prop for quick singular component styles. I'm not a fan of just extending all those props around like `p`, `px` etc. Makes components turn into a multiline mess that's 90% styles.

File: app/components/event_banner.tsx (hunk 1 above)
Comment (danielwlemmon) [reply app/components/event_banner.tsx:10]: I would be happy to dig in to Chakra if that's the direction we want to go in for the styling.

File: app/components/footer.tsx (hunk 3 above)
Comment (danielwlemmon) [reply app/components/footer.tsx:33]: Thanks for the feedback.  I wasn't sure how we wanted to structure the styling, whether it be in a separate file, inline, or in a style object like with React Native.  I just decided to put it inline for now.  From some searching it seemed that importing stylesheets with Remix wouldn't be as straightforward as with regular React, but maybe I misunderstood.

**Comment by JeffBaumgardt** [comment]
Ok going with our conversation yesterday. Lets pull styles into `.css` files and use classNames/ID's to handling the styles on the dom side.

You can follow this guide for adding css files as an import - https://remix.run/docs/en/v1/guides/styling
